lib/adapters/mydata.ts

```
@@ -24,7 +24,7 @@ interface Config {
 
 export interface DataInput {
   area: Area
-  token: string
+  token?: string
```

Token cannot be undefined and must be provided to getData/setData

We should throw an unauthorized error if token is not provided to resolvers that must be authed (everything that reads/writes in mydata)

---

Yes, thats probably for the best.